lib/index.js

```diff
@@ -280,8 +280,8 @@ function _checkTalkNamespace(title, siteInfo) {
 }
 
 function _checkMaxLength(title, namespace) {
-    const maxLength = namespace.isSpecial() ? 512 : 256;
-    if (title.length > maxLength) {
+    const maxLength = namespace.isSpecial() ? 512 : 255;
+    if (Buffer.from(title).byteLength > maxLength) {
```

Comment:
This is still incorrect. We're not specifying the encoding and I don't see a `byteLength` property in Buffer docs.

I mean literally you need to write `Buffer.byteLength(title, 'utf8')` here.

Reply:
It 'works' because byteLength is a property of [typed arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/byteLength) and Buffers are also UInt8Arrays, but point taken.